programs/ssl/ssl_server2.c

```diff
@@ -3488,18 +3498,48 @@ int main( int argc, char *argv[] )
     }
 #endif /* MBEDTLS_SSL_DTLS_HELLO_VERIFY */
 
-#if defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
+#if defined( MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED )
```

Comment:
Minor: no spaces inside defines here and in other places.
```suggestion
#if defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
```